ionic2/app/models/pokemon.ts

```diff
@@ -4,8 +4,8 @@ import { ConfigService } from '../services/config.service';
 const configServiceInjector = ReflectiveInjector.resolveAndCreate([ConfigService]);
 const config: ConfigService = configServiceInjector.get(ConfigService);
 
-export type PokemonGender = 'm' | 'f' | 'g' | 'h';
-export type PokemonAttackCategory = 'fast' | 'special';
+export type PokeGender = 'm' | 'f' | 'g' | 'h';
+export type PokeAttackCategory = 'fast' | 'special';
```

Comment:
I just noticed we never use this as far as I can see. Should we? Or should we just get rid of it?